app/src/features/admin/users/ActiveUsersList.test.tsx

```diff
@@ -63,4 +63,17 @@ describe('ActiveUsersList', () => {
       expect(getByTestId('custom-menu-button-NotApplicable')).toBeInTheDocument();
     });
   });
+
+  it('renders the button correctly', async () => {
```

Comment:
I'd be more specific: `renders the add new users button correctly`, or something like that